<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>open_window</class>
 <widget class="QMainWindow" name="open_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/bank-2.png</normaloff>icons/bank-2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(241, 242, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="openwdw_lbl_welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(15, 42, 57);</string>
    </property>
    <property name="text">
     <string>Welcome to the Hello Bank</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_CSLogin">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>201</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(211, 224, 243);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>  CUSTOMER LOGIN</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/login.png</normaloff>icons/login.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_ADLogin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(211, 224, 243);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>   ADMIN LOGIN</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/admin.png</normaloff>icons/admin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(187, 207, 237);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>    EXIT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/log-out.png</normaloff>icons/log-out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
